src/main/java/com/Proiect_Varianta_Buna/demo/Interfaces/ICustomerService.java

```diff
@@ -0,0 +1,12 @@
+package com.Proiect_Varianta_Buna.demo.Interfaces;
+
+import com.Proiect_Varianta_Buna.demo.TableEntities.Customer;
+import java.util.List;
+public interface ICustomerService {
+
+    List<Customer> getCustomers();
```

Comment:
Acest pachet de interfete ar trebui sa fie inclus in pachetul de service
+
Te rog sa folosesti DTO-uri la nivel de serviciu si sa le trimiti acelea catre Controllers (sa nu trimiti direct clase de tip  Entitate care le folosesti pentru extragerea datelor din database)

Reply:
Acest comment e valid si la restul interfetelor de servicii